test/e2e/e2e-profile/e2e-profile-test-runner/src/main/resources/getProfileAnalyzation.gql

```diff
@@ -31,7 +31,6 @@
   }",
   "variables": {
     "segmentId": "{segmentId}",
-	"start": "{start}",
-	"end": "{end}"
+    "timeRanges": [{"start": "{start}", "end": "{end}"}]
```

Comment:
We are just verifying one time range, right?

Reply:
yes, maybe I should change this to multiple time ranges verify?

Reply:
Add UTs for this seems more suitable.